gallery/src/components/NxButton/NxButtonPage.tsx

```diff
@@ -74,7 +75,8 @@ export default function NxButtonPage() {
                           id="nx-button-icon-only-example"
                           liveExample={NxButtonIconOnlyExample}
                           codeExamples={nxButtonIconOnlyCode}>
-        An example of a button containing only an icon.
+        An example of a button containing only an icon. For accessibility purposes it is important to add an
+        aria-label and aria-hidden=false for the screen reader to interpret the content correctly.
```

Comment:
If `aria-label` is added to the button rather than the icon, as you are currently doing, `aria-hidden` is not necessary right?

Reply:
yes, it is not necessary on the button itself.

Reply:
> yes, it is not necessary on the button itself.

I am awaiting a change here that reflects this.

Reply:
13478b806c6a6acf6bfdbfc6eb359746f1232792